web/src/components/tasks/CheckHousingRegistrationStatus.tsx

```diff
@@ -12,17 +12,22 @@ import { ChangeEvent, FormEvent, ReactElement, useEffect, useState } from "react
 
 interface Props {
   onSubmit: (address: HousingAddress) => void;
-  error: HotelMotelRegistrationSearchError | undefined;
+  error: HotelMotelRegistrationSearchError | MultipleDwellingSearchError | undefined;
   isLoading: boolean;
   municipalities: HousingMunicipality[];
 }
 
 const Config = getMergedConfig();
-const HotelMotelSearchErrorLookup: Record<HotelMotelRegistrationSearchError, string> = {
+const HousingRegistrationSearchErrorLookup: Record<
+  HotelMotelRegistrationSearchError | MultipleDwellingSearchError,
+  string
+> = {
   NO_PROPERTY_INTEREST_FOUND: Config.housingRegistrationSearchTask.errorTextNoPropertyInterestFound,
   NO_HOTEL_MOTEL_REGISTRATIONS_FOUND: Config.housingRegistrationSearchTask.errorTextNoHotelMotelRegistrations,
   FIELDS_REQUIRED: Config.housingRegistrationSearchTask.errorTextFieldsRequired,
   SEARCH_FAILED: Config.housingRegistrationSearchTask.errorTextSearchFailed,
+  NO_MULTIPLE_DWELLINGS_REGISTRATIONS_FOUND:
+    Config.housingRegistrationSearchTask.errorTextNoMultipleDwellingRegistrations,
 };
 
 export const CheckHousingRegistrationStatus = (props: Props): ReactElement => {
```

Comment:
Not included in the changes in this PR, but a question I had in just reviewing this file:

In the `useEffect` defined on 47, do we need the guard clauses for `communityAffairsAddress`? It looks like we're checking to see if `communityAffairsAddress` is defined and then conditionally calling `setFormValues`. Otherwise it looks like they're being initialized to undefined values.

Is there any harm in just doing this without the checks?
```typescript
setFormValues((prevValues) => {
  return {
    ...prevValues,
    address1: communityAffairsAddress.streetAddress1,
    address2: communityAffairsAddress.streetAddress2
    }
});
```

Reply:
You're totally right, this is much simpler